hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/ReconContainerMetadataManager.java

```diff
@@ -53,6 +55,17 @@ void reinitWithNewContainerDataFromOm(Map<ContainerKeyPrefix, Integer>
   void storeContainerKeyMapping(ContainerKeyPrefix containerKeyPrefix,
```

Comment:
Thanks for this comment, this is a good idea. I changed the unit tests in both the `TestReconContainerMetadataManagerImpl` and the `TestReconNamespaceSummaryManagerImpl` (and in the `TestNSSummaryTask`) to use batch version, instead of the simple store. I added the `@Deprecated` annotation to all the simple store methods.